src/components/OverlayImage.tsx

```diff
@@ -39,6 +39,28 @@ const OverlayImage = ({ url, opacity, scaling, isLocked, showDifference }: Overl
     return () => window.removeEventListener("mousemove", handleMouseMove);
   }, [mouseDown]);
 
+  // Ability to move zeplin overlay with Shift + Arrow keys
```

Comment:
I think we need to show this in the UI but not sure where. Maybe we can mention this in readme.md later.

Reply:
I was about to mention the same! Maybe a tooltip? small `?` icon somewhere that gives a list of helpful tips?

Reply:
Tooltip works but requires a bit much work for this pr. Not sure how to style it though. Another way could be showing a text info in the preview panel (at the bottom maybe) only when dragging the overlay. So it won't distract users all the time.